### ISSUE TITLE
Question about logits

### ISSUE THREAD

**Issue by jisu0123** [markus93/NN_calibration#2]
Hello.
First of all, thank you for releasing a really good code. 
I tried to run your code using the logits you uploaded. 
However, your logit files can be downloaded via git-lfs because its size is very large. 
Without purchasing, I cannot download your logit files. 

Sorry to bother you, but could you upload the files to somewhere else like google drive?
I know this is so time-consuming..

Thanks !

**Comment by kk1153** [comment]
how did you download the logits?
i get below message when i try to pull using git lfs
batch response: This repository is over its data quota. Purchase more data packs to restore access.

**Comment by jisu0123** [comment]
@kk1153 
The author of this repository provides an additional link for downloading logits. 
Please see README =)
